src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -341,6 +341,123 @@ public BitArray Not()
             return this;
         }
 
+        /*=========================================================================
+        ** Shift all the bit values to right on count bits. When count less zero it
+        ** shift all the bit values to left. New bits are be set to zero. The 
+        ** current instance is updated and returned.
+        =========================================================================*/
+        public BitArray RightShift(int count)
+        {
+            if (count <= 0)
+            {
+                if (count == 0)
+                {
+                    _version++;
```

Comment:
Yes, it is usual behavior of all BitArray methods. See Not method when len = 0 for example.